docs/config/node.rst

```diff
@@ -205,6 +208,93 @@ Ports
   range is used. If this is set to an integer value it is considered as an
   explicit single port.
 
+Advanced TCP settings
+---------------------
+
+Any interface that uses TCP (PG/HTTP/transport) shares the following settings:
+
+.. _network.tcp.no_delay:
+
+**network.tcp.no_delay**
+  | *Default:* ``true``
+  | *Runtime:* ``no``
+
+  Enable or disable the `Nagle's algorithm`_ for buffering TCP packets.
+  Buffering is disabled by default.
+
+.. _network.tcp.keep_alive:
+
+**network.tcp.keep_alive**
+  | *Default:* ``true``
+  | *Runtime:* ``no``
+
+  Configures the ``SO_KEEPALIVE`` option for sockets, which determines
+  whether they send TCP keepalive probes.
+
+.. _network.tcp.reuse_address:
+
+**network.tcp.reuse_address**
+  | *Default:* ``true on non-windows machines and false otherwise``
+  | *Runtime:* ``no``
+
+   Configures the `` SO_REUSEADDRS`` option for sockets, which determines
+   whether they should reuse the address.
+
+.. _network.tcp.send_buffer_size:
+
+**network.tcp.send_buffer_size**
+  | *Default:* ``-1``
+  | *Runtime:* ``no``
+
+  The size of the TCP send buffer. By default not explicitly set.
+
+.. _network.tcp.receive_buffer_size:
+
+**network.tcp.receive_buffer_size**
+  | *Default:* ``-1``
+  | *Runtime:* ``no``
+
+  The size of the TCP receive buffer. By default not explicitly set.
```

Comment:
same here.

Reply:
it's a hint for underlying socket. quote from docs

> In the case of stream-oriented sockets and the TCP/IP protocol, the size of the socket receive buffer may be used when advertising the size of the TCP receive window to the remote peer.

I added a link to socket docs